docs/developer-guide/local-development-gotchas.md

```diff
@@ -1,7 +1,30 @@
-# Local development gotchas
+## How do I disable Django's 'debug' mode?
 
-This page lists common local development issues and how do resolve them.
+Sometimes you need to disable Django's DEBUG mode in order to test certain behaviours. To do this, you have two options:
```

Comment:
I think the original introductory text here is also useful

Reply:
Done, kept original text